src/api/Utils.test.js

```diff
@@ -0,0 +1,38 @@
+import mockAxios from 'axios';
+import APIUtils, { useLocalData } from '../api/Utils';
+
+const apiUtils = new APIUtils();
+
+describe('Tests Utils module', () => {
+  let waveformUrl;
+  beforeAll(() => {
+    waveformUrl = useLocalData
+      ? 'http://localhost:3123/data/mock-response-waveform.json'
+      : 'https://spruce.dlib.indiana.edu/master_files/j3860704z/waveform.json';
```

Comment:
So typically we'll want to mock all API calls, and never reach out to a real server in tests.   Does Axios provide a way to reach out to a real server?   What does it recommend?

Reply:
You mean in tests to reach to a real server? The idea is to not to test axios but to test the results of the API calls in our app. So we just mimic the output of axios in the test without having to actually going to a real server. 

Did I answer your question? :D

And this test is misleading because it doesn't test anything and I wrote this to learn mocking third party libraries.

Reply:
Yea, I think we're on the same page.  Do we need this test still?

Reply:
If so, maybe to make it clear that that we're mocking network requests, on line #9 we just do something like:

`waveformUrl = 'http://mockurl.com' ?   And probably can lose the `beforeAll()`?

Reply:
@adamjarling No we don't need this test. Should I remove this?
Yeah, if we are going to keep it, that change would make things clearer.

Reply:
Let's keep the test, just for a reference moving forward.  How about you put a comment in the test, noting it's for reference, and also make the URL adjustment?   Then let's merge it.